File: 2-Semestre/Taller de Programación/Personal/Python-C#/Ejercicio 3.cs
/*
Conversión de una longitud dada en pies a centímetros.
Los pies son ingresados por el usuario. Considere que: 1 pie = 30.48 centímetros.
*/

using System;

namespace Ejercicio_3
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Este programa convierte pies a centimetros\n");

            double ft;
            do
            {
                Console.Write("Ingrese la longitud en pies: ");
                ft = Convert.ToDouble(Console.ReadLine());

                if (ft <= 0)
                {
                    Console.WriteLine("[!] La longitud no puede ser menor o igual a 0.");
                }
            } while (ft <= 0);

            double cm = ft * 30.48;
            Console.WriteLine($"La longitud en centimetros es de {cm} cm");
        }
    }
}

File: 1-Semestre/Desarrollo de algoritmos/Actividades/Diagramas de flujo-Python (Semana 5)/Ejercicio 3.py
'''
Conversión de una longitud dada en pies a centímetros.
Los pies son ingresados por el usuario. Considere que: 1 pie = 30.48 centímetros.
'''

print("Este programa convierte pies a centimetros")
ft = int(input("Ingrese la longitud en pies: "))
cm = 0

while ft <= 0:
    print("[!] La longitud no puede ser menor o igual a 0.")
    ft = int(input("Ingrese la longitud en pies: "))

cm = ft * 30.48
print(f"La longitud en centimetros es de {cm} cm")